packages/node-opentelemetry/src/otel.ts

```diff
@@ -41,6 +45,7 @@ export type SDKConfig = {
   consoleCapture?: boolean;
   experimentalExceptionCapture?: boolean;
   instrumentations?: InstrumentationConfigMap;
+  programmaticImports?: boolean; // TEMP
```

Comment:
nit: i know this is temporary, but i feel like dynamicImports makes more sense than programmatic?

Reply:
hmm...its not dynamic imports technically